components/user/settings/settingsSection.js

```diff
@@ -4,30 +4,48 @@ import Section from '../../common/section'
 import Card from '../../common/card'
 import UnderlinedHeading from '../../common/underlinedHeading'
 
-const UserSettingsSection = ({ ...props }) => (
-  <Section
-    display='flex'
-    justifyItems='center'
-    flexDirection='column'
-    alignItems='center'
-    padding={{ base: '3rem 1.5rem', lg: '6rem 7.5rem' }}
-    backgroundColor='white'
-    {...props}
-  >
-    <UnderlinedHeading text='User Settings' align='center' marginBottom='3rem' />
-    <Card shadowSz='lg' w='100%' maxW='45rem'>
-      <Text
-        textTransform='uppercase'
-        textAlign={{ base: 'center', lg: 'left' }}
-        color='ocean'
-        fontWeight='bold'
-        fontSize='0.875rem'
-        marginBottom='2.5rem'
-      >
-        Settings
-      </Text>
-    </Card>
-  </Section>
-)
+import { useAuth } from '../../../utils/useAuth'
+
+const UserSettingsSection = ({ ...props }) => {
+  const { user } = useAuth()
+
+  return (
+    <Section
+      display='flex'
+      justifyItems='center'
+      flexDirection='column'
+      alignItems='center'
+      padding={{ base: '3rem 1.5rem', lg: '4rem 7.5rem' }}
+      backgroundColor='white'
+      {...props}
+    >
+      <UnderlinedHeading text='User Settings' align='center' marginBottom='3rem' />
+      <Card shadowSz='lg' w='100%' maxW='80%'>
+        <Text
+          textTransform='uppercase'
+          textAlign={{ base: 'center', lg: 'left' }}
+          color='ocean'
+          fontWeight='bold'
+          fontSize='0.875rem'
+          marginBottom='1.5rem'
+        >
+          Billing info
+        </Text>
+        <Text>
+          Email
+        </Text>
+        <Text>
+          {user.email}
+        </Text>
```

Comment:
We're not allowing users to change their email address, right? If we're not allowing that, I wonder if there's a point in putting this on the settings page? Settings indicate something that can be changed.

Reply:
Sounds good I was just showing some user data to get a feel for it. Removed

Reply:
Yeah.I was just wondering. :) Didn't want users to get confused or frustrated if we displayed something that couldn't be changed.